batch/batch/utils.py

```diff
@@ -194,6 +209,36 @@ def is_valid_cores_mcpu(cores_mcpu: int):
     return quarter_cores & (quarter_cores - 1) == 0
 
 
+def resources_to_machine_type(cores_mcpu, memory_bytes, preemptible):
```

Comment:
I don't think we should automatically convert pool jobs into private jobs, so I think this function is unnecessary.